test/spec/adapters/adbund_spec.js

```diff
@@ -0,0 +1,102 @@
+import { expect } from 'chai';
+import Adapter from '../../../src/adapters/adbund';
+import adapterManager from 'src/adaptermanager';
```

Comment:
import of adaptermanager is causing the problem. Remove this line. 
You can check https://github.com/prebid/Prebid.js/blob/master/test/spec/adapters/hiromedia_spec.js